ui/pages/send/send-header/send-header.stories.js

```diff
@@ -17,40 +16,44 @@ import SendHeader from './send-header.component';
 export default {
   title: 'Pages/Send/SendHeader',
   id: __filename,
+  argsTypes: {
+    asset: { control: 'select' },
+    stage: { control: 'select' },
+  },
 };
 
-export const DefaultStory = () => {
-  const store = createStore(
-    combineReducers({ send: sendSBReducer, history: historySBReducer }),
-  );
-  const state = store.getState();
-  const { send } = state;
-  const asset =
-    select('Asset', [ASSET_TYPES.NATIVE, ASSET_TYPES.TOKEN]) || send.asset;
-
-  const stage =
-    select('Stage', [
-      SEND_STAGES.ADD_RECIPIENT,
-      SEND_STAGES.DRAFT,
-      SEND_STAGES.EDIT,
-      SEND_STAGES.INACTIVE,
-    ]) || send.stage;
+const store = createStore(
+  combineReducers({ send: sendSBReducer, history: historySBReducer }),
+);
+const state = store.getState();
+const { send } = state;
 
+export const DefaultStory = (args) => {
   useEffect(() => {
-    store.dispatch(updateSendAsset(asset));
-  }, [store, asset]);
+    store.dispatch(updateSendAsset(args.asset));
+  }, [args.asset]);
 
   useEffect(() => {
-    store.dispatch(updateSendStage(stage));
-  }, [store, stage]);
+    store.dispatch(updateSendStage(args.stage));
+  }, [args.stage]);
 
   return (
     <Provider store={store}>
       <div style={{ width: 600 }}>
-        <SendHeader />
+        <SendHeader {...args} />
       </div>
     </Provider>
   );
 };
 
 DefaultStory.storyName = 'Default';
+DefaultStory.args = {
+  asset: [ASSET_TYPES.NATIVE, ASSET_TYPES.TOKEN] || send.asset,
+  stage:
+    [
+      SEND_STAGES.ADD_RECIPIENT,
+      SEND_STAGES.DRAFT,
+      SEND_STAGES.EDIT,
+      SEND_STAGES.INACTIVE,
+    ] || send.stage,
+};
```

Comment:
It would be good if the functionality of the knobs still worked. You can compare [the current version of SendHeader story](https://metamask.github.io/metamask-storybook/index.html?path=/story/ui-pages-send-send-header-send-header-stories-js--default-story&knob-Advanced%20Inline%20Gas=true&knob-Asset=NATIVE&knob-Customize%20Transaction%20Nonce=true&knob-Show%20Hex%20Data=true&knob-Stage=ADD_RECIPIENT&knob-Warning=Something%20else)


Uploading sendheader.mp4…

Reply:
@georgewrmarshall With this changes functionality of the knobs still worked.
You can see changes here: 3123dbb12e2dd7ae6aea5e4bc5b0b6e4f23aab41
Is it okay now?

Reply:
Looks good!!